examples/blockly-umd/index.js

```diff
@@ -23,7 +23,75 @@
 
 document.addEventListener("DOMContentLoaded", function () {
 
-    var toolbox = document.getElementById('toolbox');
+    const toolbox = {
```

Comment:
Nit: Indentation should only be by two spaces for things that aren't line continuations. (You may want to check your IDE settings for a way to modify this)

Please change this to  look like:

```
const toolbox = {
  kind: 'flyoutToolbox',
  contents: [ /* etc... */ ],
}
```